client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/style.scss

```diff
@@ -410,11 +414,11 @@ $design-button-primary-color: rgb( 17, 122, 201 );
 			right: 0;
 			transform: translateY( 118px );
 			transition: opacity 0.15s ease-in-out, transform 0.15s ease-in-out;
+			z-index: 1;
 
 			&.is-visible {
 				opacity: 1;
 				transform: translateY( 0 );
-				z-index: 1;
```

Comment:
@taipeicoder Do you have any concerns to set the z-index when it's visible? If we only apply the z-index when it's visible, we won't see the slide-out animations and it will disappear immediately as it's covered by the previews.

Reply:
Just tested and didn't break anything. I think I just added z-index: 1 only when visible because the footer would block the thumbnail section. But since we're using `pointer-events` to solve that, we can always have the footer as z-index: 1.